pettingzoo/classic/rlcard_envs/texas_holdem.py

```diff
@@ -46,7 +48,12 @@ class raw_env(RLCardBase):
     def __init__(self, num_players=2):
         super().__init__("limit-holdem", num_players, (72,))
 
-    def render(self, mode="human"):
+    def render(
+        self,
+        mode="human",
+        most_recent_move: Optional[Dict] = None,
+        win_message: Optional[str] = None,
+    ):
```

Comment:
This shouldn't be in render to keep inline with the other environments, could it perhaps be in the init for the environment?